content/mobile-tor/contents.lr

```diff
@@ -0,0 +1,222 @@
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-provided-a-bridge.png" alt="Provide brigde addresses on Tor Browser for Android">
+
+
+### MANAGING IDENTITIES
+
+#### New Identity
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-new-identity.png" alt="New Identity on Tor Browser for Android">
+
+When Tor Browser is running, you would see so in your phone's notification panel along with the button "NEW IDENTITY".
+Clicking on this button will provide you with a new identity.
+Unlike in Tor Browser for Desktop, the "NEW IDENTITY" button in Tor Browser for Android does not prevent your subsequent browser activity from being linkable to what you were doing before.
+Selecting it will only change your Tor circuit.
+
+### SECURITY SETTINGS
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-security-settings.gif" alt="Security settings and security slider on Tor Browser for Android">
+
+[Security settings](https://tb-manual.torproject.org/security-settings/) disable certain web features that can be used to attack your security and anonymity.
+Tor Browser for Android provides the same three security levels that are available on desktop.
+You can modify the security level by following given steps:
+
+* Tap on a button of 3 vertical dots in URL bar.
+* Scroll down and tap on "Security Settings".
+* You can now select an option from the security level slider.
+
+
+### UPDATING
+Tor Browser must be kept updated at all times. If you continue to use an outdated version of the software, you may be vulnerable to serious security flaws that compromise your privacy and anonymity.
+You can update Tor Browser automatically or manually.
+
+#### Updating Tor Browser for Android automatically
+This method assumes that you have either Google Play or F-droid installed on your mobile device.
+
+##### Google Play
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-update-google-play.png" alt="Updating Tor Browser for Android on Google Play">
+
+Click on the hamburger menu next to the search bar and navigate to "My apps & games" > "Updates".
+If you find Tor Browser on the list of apps which need updating, select it and click the "Update" button.
+
+##### F-Droid
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-update-f-droid.png" alt="Updating Tor Browser for Android on F-droid">
+
+Click on "Settings", then go to "Manage installed apps".
+On the next screen, select Tor Browser and finally click on the "Update" button.
+
+#### Updating Tor Browser for Android manually
+
+With Tor Browser closed, remove it from your system by uninstalling it using your device's settings.
+Depending on your mobile device's brand, navigate to Settings > Apps, then select Tor Browser and click on the "Uninstall" button.
```

Comment:
Is uninstalling the app necessary? I thought you can download the new version and upgrade the app in-place. You can also download the apk using Tor Browser (by going to the website as described below), and then install it. If it is working correctly, Tor Browser can install the update itself - but that may be broken right now.